examples/container-with-vitest/test/ReactWrapper.test.ts

```diff
@@ -0,0 +1,20 @@
+import { experimental_AstroContainer as AstroContainer } from 'astro/container';
+import { expect, test } from 'vitest';
+// @ts-ignore
+import ReactWrapper from '../src/components/ReactWrapper.astro';
```

Comment:
Was there an error here? This is supported in our TypeScript plugin

Reply:
Yes: https://github.com/withastro/astro/pull/11051#discussion_r1601620312